clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -203,6 +197,14 @@ export class SettingsService {
         dialectType,
       ),
     ];
+    paths.forEach((localPath) => {
+      const wsFolders = SettingsUtils.getWorkspaceFoldersPath(true);
+      wsFolders.forEach((wsFolder) => {
+        paths.push(path.join(wsFolder, localPath));
```

Comment:
Doesn't this produce garbage when localPath is absolute?

Reply:
yes it does , however it is a necessity for being backward compatible.

Reply:
How about checking if the path is absolute and then not produce the garbage values?

Reply:
resolved at : 64eab1d5606e58ec43906144b329e4bcd9e7801c

Reply:
So now, the absolute paths are completely ignored... does not seem resolved to me.

Reply:
Doesn't seems they are ignored ? 

![image](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/assets/104970275/ad513fac-7b3d-4b8e-96db-6e65f0ef3509)
![image](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/assets/104970275/a678be2e-f05a-4bf3-8503-abd33a4a0c2d)

Reply:
I missed that you are modifying the path array... that being said, the unexpanded relative paths remain in the array, their expanded versions are pushed back (changing the original search order).
And then you end up interpreting the relative paths in the context of the new extension directory  in `searchCopybookInExtensionFolder`.

Reply:
sure, this : 8b04b1878af8104b3e0fcc4651bf598d9a55ee42 should solve it.